portal/templates/portal/users/detail.html

```diff
@@ -104,7 +105,12 @@ <h1 class="title is-1">{{ user.full_name }}</h1>
 
 <section class="section">
     <div class="container">
-        <p class="has-text-grey">Coming soon...</p>
+        {% if request.user.is_superuser or request.user == user and user.is_rpi%}
+        <div class="block">
+            <a href="{% url 'user_attendance' user.pk %}{% active_semester_query %}" class="button">Attendance</a>
+        </div>
+        {% endif %}
+        <p class="has-text-grey">More coming soon...</p>
```

Comment:
what will go here?

Reply:
- [ ] a list of the semesters that the student have been enrolled in, and for each semester:
  - [ ] their team